mypy/fastparse.py

```diff
@@ -1205,7 +1205,10 @@ def visit_Attribute(self, n: Attribute) -> Union[MemberExpr, SuperExpr]:
     def visit_Subscript(self, n: ast3.Subscript) -> IndexExpr:
         e = IndexExpr(self.visit(n.value), self.visit(n.slice))
         self.set_line(e, n)
-        if isinstance(e.index, SliceExpr):
+        if (
+            isinstance(e.index, SliceExpr) or
+            (sys.version_info <= (3, 9) and isinstance(e.index, TupleExpr))
```

Comment:
Does this mean that the version check is only true for versions earlier than 3.9.0 (since `version_info` has extra components beyond the first three) or up to 3.9.0? Please add a comment about this.

Or could you perhaps use `< (3, 9)` as the condition?

Reply:
Thanks! I did mean that. Changed a couple things and added comments; hopefully the result is more understandable.